crew

```diff
@@ -27,6 +27,9 @@ else
   CREW_NPROC = ENV["CREW_NPROC"]
 end
 
+# Set CREW_NOT_COMPRESS from environment variable
+CREW_NOT_COMPRESS = ! File.exist?("#{CREW_PREFIX}/bin/compressdoc") || ENV["CREW_NOT_COMPRESS"]
```

Comment:
Since I don't prefer that way, installing `compressdoc` automatically.  It's user's choice.

Regarding to making `crew compress` command, I like it.  I would like to upgrade filelist also if manual pages are compressed, but it's too much work to do.  So, I'm not asking you to do so.  I also like to have `crew shrink`.